knockserver/models.py

```diff
@@ -3,7 +3,31 @@
 from knockserver.database import Base
 
 class ExampleObject(Base):
```

Comment:
@ajd4223 Could you explain what the `ExampleObject` is for? Or is it just from the original sample project?